classlib/src/main/java/org/teavm/classlib/java/util/CollectionsFactory.java

```diff
@@ -0,0 +1,92 @@
+/*
+ *  Copyright 2014 Alexey Andreev.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.teavm.classlib.java.util;
+
+import static org.teavm.classlib.java.util.TObjects.requireNonNull;
+import org.teavm.classlib.java.lang.TIllegalArgumentException;
+import org.teavm.classlib.java.lang.TNullPointerException;
+
+/**
+ * Factory-methods for List/Set/Map.of(...).
+ */
+class CollectionsFactory {
+
+    private CollectionsFactory() {
+    }
+
+    /**
```

Comment:
Is there real need in such obvious doc comments? Is there any purpose on doc comments on non-public code?

Reply:
I write a lot of library code in our company -- I guess I'm just used to document everything... but I can remove these.